protocol/Protocol.py

```diff
@@ -3134,7 +3152,7 @@ def getBanDB(self):
 	assert(not p._validChannelSyntax("#abcde")[0])
 	assert(not p._validChannelSyntax("ab cde")[0])
 
-	minver = "104.0"
+	p._root.min_spring_version = "104.0"
```

Comment:
why this change? in this case "104.0" is specific for the tests.

Reply:
Because I thought it was cleaner not to have to pass the current min version in to _validEngineVersion, and make _validEngineVersion a method of the procotol class.

The root here is a dummy, so afaics there is nothing odd about this as unit test code.

Reply:
Re-open if you disagree!

Reply:
As I understand it, 104.x doesn't run on Mac, while 103.x does. If I'm not wrong, I can understand why people might complain with enforcing 104.x

Reply:
we need to move forward... apple has to decided to deprecate opengl. very likely newer version of osx will break spring 103.x, too.

Reply:
This piece of code is a unit test - not a hardcoded default. (I don't see any future worth the trouble for macs, though.)

Reply:
I see - my bad for looking at just code snippets.

Reply:
@gajop
deprecating usually means that it will be removed after some time.

also 104.0 is currently planned as min version, so your concerns are valid.